modules/flowable-engine-common/src/main/java/org/flowable/common/engine/impl/scripting/ScriptingEngines.java

```diff
@@ -166,14 +178,18 @@ protected ScriptEngine getEngineByName(String language) {
         return scriptEngine;
     }
 
-    /** override to build a spring aware ScriptingEngines */
+    /**
+     * override to build a spring aware ScriptingEngines
+     */
     protected Bindings createBindings(VariableContainer variableContainer) {
         return scriptBindingsFactory.createBindings(variableContainer);
     }
 
-    /** override to build a spring aware ScriptingEngines */
-    protected Bindings createBindings(VariableContainer variableContainer, boolean storeScriptVariables) {
```

Comment:
Wasn't sure how backwards compatible we have to be here, as is an "impl" package. Will add it back

Reply:
Yes the `ScriptingEngines` are in an `impl` class. However, they are something that could have been extended by someone to provide a custom scripting functionality. That is why the methods with name `evaluateWithEvaluationResult` were added like that instead of changing the methods named `evaluate`